package.xml

```diff
@@ -1,9 +1,25 @@
 <?xml version="1.0"?>
 <package>
-    <name>robotont_description</name>
-    <version>0.0.5</version>
-    <description>The robotont_description package</description>
-    <maintainer email="madis.kaspar.nigol@ut.ee">Madis Kaspar Nigol</maintainer>
-    <license>Apache 2.0</license>
-    <buildtool_depend>catkin</buildtool_depend>
+  <name>robotont_description</name>
+  <version>0.0.5</version>
+  <description>The robotont_description package</description>
+
+  <author>Madis Kaspar Nigol</author>
+
+  <maintainer email="veiko.vunder@ut.ee">Veiko Vunder</maintainer>
+  <maintainer email="karl.kruusamae@ut.ee">Karl Kruusamäe</maintainer>
+
+  <url type="website">https://wiki.ros.org/robotont</url>
+  <url type="website">http://robotont.ut.ee</url>
+  <url type="repository">http://github.com/robotont/robotont_description</url>
+
+  <license>Apache 2.0</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
```

Comment:
@Veix123 kas meil peaks olema siin ka `<depend>xacro</depend>`?

Reply:
Resolved in 05701491c9424b33d3c78da4ef675ce381b0d53b. Võtsin eeskuju [ur_description](https://github.com/ros-industrial/universal_robot/tree/kinetic-devel/ur_description) kimbust ja [siit](https://answers.ros.org/question/315252/are-urdf-rviz-and-xacro-packages-required/).